src/Thentos/Backend/Api/Adhocracy3.hs

```diff
@@ -345,6 +346,12 @@ login r = AC.logIfError'P $ do
 
 -- * aux
 
+-- | Render Thentos/A3-specific custom headers using the names expected by A3.
+renderA3HeaderName :: RenderHeaderFun
```

Comment:
The other function is called `renderThentosHeaderName`, I just adapted that name. Since either function alway generates a single header name, I guess it's OK.